docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -41,7 +41,7 @@ The following table lists browsers that can be detected automatically.
 Browser                                      | Browser Alias
 ---------------------------------------------| -------------------
 Chromium                                     | `chromium`
-Google Chrome (Stable, Beta, Dev channel)    | `chrome`
+Google Chrome (Regular Channels)             | `chrome`
```

Comment:
Is it a common thing to call these channels `regular` in contrast to the `Canary` channel?